cypress/integration/home_page_spec.js

```diff
@@ -0,0 +1,50 @@
+import { aliasQuery, hasOperationName } from '../utils/graphql-test-utils';
+
+describe('Not Fur-gotten Home Page User Flow', () => {
+  beforeEach(() => {
+    cy.intercept(
+      {
+        method: 'POST',
+        url: 'https://not-fur-gotten-be.herokuapp.com/graphql',
+        headers: {
+          'x-gql-operation-name': 'getAllPets',
```

Comment:
I'm glad we were able to go back and implement this middle-ware to make this work, awesome job!